File: agafonov/diagrams/src/com/company/route/Route.java
package com.company.route;

public class Route {
    private Station[] stations;

    public Route(){}
    public Route(Station[] stations){
        this.setStations(stations);
    }

    public Station[] getStations() {
        return stations;
    }

    public void setStations(Station[] stations) {
        this.stations = stations;
    }


}

File: UML_vantuz/FIrstUMLLab/FIrstUMLLab/Route.cs
using System;
using System.Collections.Generic;

namespace FIrstUMLLab
{
    class Route
    {
        public int Number { get; private set; }
        public Transport Transport { get; private set; }
        public List<Station> Stations { get; private set; }

        public Route(int num, Transport transport, List<Station> stations)
        {
            this.Number = num;
            this.Transport = transport;
            this.Stations = stations;
        }

        public double Driving()
        {
            return 0;
        }
    }
}
